spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/main/java/org/springframework/boot/configurationprocessor/TypeUtils.java

```diff
@@ -142,8 +142,7 @@ public String getJavaDoc(Element element) {
 		String javadoc = (element != null
 				? this.env.getElementUtils().getDocComment(element) : null);
 		if (javadoc != null) {
-			javadoc = javadoc.replaceAll("\\n", "");
-			javadoc = javadoc.trim();
```

Comment:
I still would like surrounding spaces to be removed. Do you disagree with that?

Reply:
@snicoll No, I just collapsed the two statements into one. I thought it's less verbose than before.

Reply:
I missed that 😓